src/index.html

```diff
@@ -9,6 +9,30 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Enclosures</h1>
+  <div id="qa-block" class=" block block-upper">
```

Comment:
what is space doing there?
```suggestion
  <div id="qa-block" class="block block-upper">
```